taskipy/pyproject.py

```diff
@@ -10,29 +10,47 @@
     MissingPyProjectFileError,
     MissingTaskipyTasksSectionError
 )
+from taskipy.variable import Variable
 
 
 class PyProject:
     def __init__(self, base_dir: Path):
         pyproject_path = self.__find_pyproject_path(base_dir)
-        self.__items = PyProject.__load_toml_file(pyproject_path)
+        self.__items = self.__load_toml_file(pyproject_path)
 
     @property
     def tasks(self) -> Dict[str, Task]:
         try:
-            return {
-                task_name: Task(task_name, task_toml_contents) for
-                task_name, task_toml_contents in self.__items['tool']['taskipy']['tasks'].items() }
+            toml_tasks = self.__items['tool']['taskipy']['tasks'].items()
         except KeyError:
             raise MissingTaskipyTasksSectionError()
 
+        tasks = {}
+        for name, toml_contents in toml_tasks:
+            tasks[name] = Task(name, toml_contents)
+
+        return tasks
+
     @property
-    def variables(self) -> Dict[str, Task]:
+    def variables(self) -> Dict[str, Variable]:
         try:
-            return self.__items['tool']['taskipy'].get('variables', {})
+            toml_vars = self.__items['tool']['taskipy'].get('variables', {})
         except KeyError:
             return {}
 
+        vars_dict = {}
```

Comment:
```suggestion
        vars_dict: Dict[str, Variable] = {}
```
This will save any future devs from mistakenly putting anything that is not `Variable` in the dict :)

Reply:
Sounds good to me! If you have a better name than `vars_dict`, I'm happy to hear it too :smile: Needed to avoid the build-in `vars` method name.

Reply:
Maybe `loaded_vars`? Works either way tho